staging/src/k8s.io/client-go/tools/remotecommand/v1.go

```diff
@@ -56,7 +56,6 @@ func (p *streamProtocolV1) stream(conn streamCreator) error {
 		defer klog.V(6).Infof("Done copying %s", s)
 		if _, err := io.Copy(dst, src); err != nil && err != io.EOF {
 			klog.Errorf("Error copying %s: %v", s, err)
-			errorChan <- err
```

Comment:
don't we need to keep sending the error?

Reply:
This was a previous red herring. I don't think we should modify the `v1.go` file.

Reply:
There are currently no modifications to this file (`v1.go`) in the PR (which is the way it should be).

Reply:
we can resolve this